### PR TITLE
Add a newline and link to Hosting section

### PR DESCRIPTION
**Pull request by Nazar-Ant**

### CODE REVIEW

**Review by KnorpelSenf — APPROVED** [review]
Thanks!

**Comment by KnorpelSenf** [comment]
Let's wait for @roj1512 to decide if it's ready. If he has comments, it'd be too early to translate

**Comment by Nazar-Ant** [comment]
@KnorpelSenf That is, labels for such PRs are added only after agreement? OK, I'll know.

**Review by roj1512 — COMMENTED** [review]
https://t.me/grammyjs/116169

**Comment by KnorpelSenf** [comment]
> @KnorpelSenf That is, labels for such PRs are added only after agreement? OK, I'll know.

Yep :)

**Comment by KnorpelSenf** [comment]
Only ready for translation actually. You can add 🇺🇦 yourself as soon as some Ukrainian translation is completed. If something isn't translatable, you can also add this label yourself.

File: site/docs/guide/introduction.md [hunk 1]
```diff
@@ -112,7 +112,8 @@ If you don't have much experience with programming yet, **it makes sense to star
 
 Even if you have written code for Node.js before, you should consider giving Deno a go.
 Many things that are hard under Node.js are a no-brainer under Deno.
-Deno
+
+Deno:
```
Comment (KnorpelSenf) [inline site/docs/guide/introduction.md]:
```suggestion
Deno
```

**Comment by Nazar-Ant** [comment]
Added a new commit. Did I do the right thing about the commit?

**Comment by KnorpelSenf** [comment]
Yes, thanks. My comment was just late. @roj1512 will approve soon and then translation can happen.

**Review by roj1512 — APPROVED** [review]
LGTM. Thank you so much!

**Review by CikiMomogi — APPROVED** [review]

**Comment by habemuscode** [comment]
https://github.com/grammyjs/website/pull/641

**Comment by habemuscode** [comment]
https://github.com/grammyjs/website/pull/646

**Review by fadzikri — COMMENTED** [review]
OK

**Pull request merged**